cucumber-junit-platform-engine/README.md

```diff
@@ -266,9 +266,39 @@ Feature: Isolated scenarios
 
 with this configuration:
 
+
 ```properties
 cucumber.execution.exclusive-resources.isolated.read-write=org.junit.platform.engine.support.hierarchical.ExclusiveResource.GLOBAL_KEY
 ```
+### Sequential Scenario Execution With Parallel Run
```

Comment:
This section and its fails to mention the most important aspect first: JUnit 4 compatibility. Please rewrite with this in mind.